snapcraft/internal/lxd.py

```diff
@@ -147,3 +134,47 @@ def _wait_for_network(self):
                 if retry_count == 0:
                     raise e
         logger.info('Network connection established')
+
+
+def _get_default_remote():
+    """Query and return the default lxd remote.
+
+    Use the lxc command to query for the default lxd remote. In most
+    cases this will return the local remote.
+
+    :returns: default lxd remote.
+    :rtype: string.
+    :raises snapcraft.internal.errors.SnapcraftEnvironmentError:
+        raised if the lxc call fails.
+    """
+    try:
+        default_remote = check_output(['lxc', 'remote', 'get-default'])
```

Comment:
Since I am not redirecting stderr it should be printed out; but if this command fails it probably means you need to go through the setup anyways.